enterprise/frontend/src/embedding-sdk/hooks/private/use-sdk-element-size.ts

```diff
@@ -0,0 +1,7 @@
+import { useElementSize } from "@mantine/hooks";
+
+export const useSdkElementSize = () => {
+  const { height, ref, width } = useElementSize();
+
+  return { height: height || (width ? width / 9 : 500), ref, width };
```

Comment:
What's `|| (width ? width / 9 : 500)`? :mage_man:

Reply:
Magic!

Reply:
I'll add a comment or function or something here - basically we should make sure that our elements actually have some sort of height/width, and it's been an issue with some of our components not following the div size/layout in client apps. So this is more of an attempt than anything but it seems to be working